impl/src/main/java/org/ehcache/impl/internal/events/CacheEventDispatcherFactoryImpl.java

```diff
@@ -63,7 +63,7 @@ public void stop() {
   @Override
   public <K, V> CacheEventDispatcher<K, V> createCacheEventDispatcher(Store<K, V> store, ServiceConfiguration<?>... serviceConfigs) {
     String threadPoolAlias = defaultThreadPoolAlias;
-    DefaultCacheEventDispatcherConfiguration config = findSingletonAmongst(DefaultCacheEventDispatcherConfiguration.class, (Object[]) serviceConfigs);
+    DefaultCacheEventDispatcherConfiguration config = findSingletonAmongst(DefaultCacheEventDispatcherConfiguration.class, serviceConfigs);
```

Comment:
You sure this (and the other `Object[]` cast removals) doesn't trip a compiler warning around the varargs call?

Reply:
Arg... They were not showing at first and I was surprised. But they are there indeed. Rollbacking.